### PR TITLE
Add Unstoppable Domains

### PR DESCRIPTION
**Pull request by jkmartindale**
Currently Alexa # 10,909, <https://unstoppabledomains.com>

### CODE REVIEW

**Comment by ApCoder123** [comment]
Since there is no documentation available, please could you provide us with screenshots of the setup/login process as evidence of 2FA? Please remember to block out any personal information.

File: entries/u/unstoppabledomains.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Unstoppable Domains": {
+    "domain": "unstoppabledomains.com",
+    "img": "unstoppabledomains.com.png",
```
Comment (ApCoder123) [inline entries/u/unstoppabledomains.com.json]: Please use this SVG version of the logo instead:
```suggestion
```

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Comment by jkmartindale** [comment]
image  
image  
image  
image  
image  
image

**Review by ApCoder123 — APPROVED** [review]
Thanks!

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**